server/controllers/usersController.js

```diff
@@ -50,5 +50,44 @@ class UsersController {
 			return error;
 		}
 	}
+	static async login(req, res) {
+		try {
+			const { email, password } = req.body;
+			const userLogin = await UserModel.login(email);
+			const {
+				firstName,
+				lastName,
+				nationality,
+				telephone,
+				workAddress,
+				isAdmin
+			} = userLogin;
+			if (userLogin) {
+				if (Auth.comparePassword(password, userLogin.password)) {
+					const token = Auth.generateToken(userLogin.isadmin, userLogin.email);
+					return res.status(statusCode.STATUS_OK).send({
+						status: statusCode.STATUS_OK,
+						data: {
+							token,
+							firstName: userLogin.firstname,
+							lastName: userLogin.lastname,
+							email: userLogin.email,
```

Comment:
Since you have destructured all the fields, you should use them instead of having to type `userLogin.firstname`